core/src/main/resources/default-configs/default-jgroups-azure.xml

```diff
@@ -5,6 +5,7 @@
 <config xmlns="urn:org:jgroups"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/jgroups-4.2.xsd">
+   <!-- jgroups.tcp.address is deprecated and will be removed, see ISPN-11867 -->
    <TCP bind_addr="${jgroups.bind.address,jgroups.tcp.address:SITE_LOCAL}"
         bind_port="${jgroups.bind.port,jgroups.tcp.port:7800}"
```

Comment:
How about `jgroups.tcp.port`. Should we remove as well ?

Reply:
There are 2 problems with `jgroups.tcp.address`:
1. It's reused by other protocols in the stack, which may or may not use TCP sockets
2. It's arbitrary, there's no reason someone would want a different IP address for TCP vs UDP.

`jgroups.tcp.port` doesn't have any of those problems: each protocol using TCP sockets directly must have a different port, so the system property cannot be reused.